src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.cs

```diff
@@ -11,36 +11,36 @@ public partial class LabelHandlerTests : HandlerTestBase<LabelHandler>
 		[Fact]
 		public async Task BackgroundColorInitializesCorrectly()
 		{
-			var label = new LabelStub()
+			var labelStub = new LabelStub()
 			{
 				BackgroundColor = Color.Blue,
 				Text = "Test"
 			};
 
-			await ValidateNativeBackgroundColor(label, Color.Blue);
+			await ValidateNativeBackgroundColor(labelStub, Color.Blue);
 		}
 
 		[Fact]
 		public async Task TextInitializesCorrectly()
 		{
-			var label = new LabelStub()
+			var labelStub = new LabelStub()
 			{
 				Text = "Test"
 			};
 
-			await ValidatePropertyInitValue(label, () => label.Text, GetNativeText, label.Text);
+			await ValidatePropertyInitValue(labelStub, () => labelStub.Text, GetNativeText, labelStub.Text);
 		}
 
 		[Fact]
 		public async Task TextColorInitializesCorrectly()
 		{
-			var label = new LabelStub()
+			var labelStub = new LabelStub()
 			{
 				Text = "Test",
 				TextColor = Color.Red
 			};
 
-			await ValidatePropertyInitValue(label, () => label.TextColor, GetNativeTextColor, label.TextColor);
+			await ValidatePropertyInitValue(labelStub, () => labelStub.TextColor, GetNativeTextColor, labelStub.TextColor);
 		}
```

Comment:
Should the text alignment tests be added here too? (And not just in the .Android and .iOS test files?)